### ISSUE TITLE
Change the way resources yaml file is being loaded

### ISSUE THREAD

**Issue by zroubalik** [kedacore/keda-olm-operator#106]
At the moment the [resources file](https://github.com/kedacore/keda-olm-operator/blob/main/resources/keda.yaml ) is being loaded into the operator during the startup: https://github.com/kedacore/keda-olm-operator/blob/cbd80d4af6e95d5f0ad36af6631d3a8a008c5e3e/controllers/keda/kedacontroller_controller.go#L76-L83

It would be better to use go embed feature introduced in Go 1.16: https://golangtutorial.dev/tips/embed-files-in-go/

**Comment by husnialhamdani** [comment]
Hi @zroubalik,
Can I work on this?

**Comment by zroubalik** [comment]
@husnialhamdani absolutely, please coordinate with @joelsmith, he is the main point of contact for this repo.

**Comment by joelsmith** [comment]
@husnialhamdani, that would be great, thanks! Let me know if you have any questions along the way

**Comment by linkvt** [comment]
Implemented in #278
